packages/block-editor/src/components/text-decoration-control/stories/index.story.js

```diff
@@ -8,26 +8,65 @@ import { useState } from '@wordpress/element';
  */
 import TextDecorationControl from '../';
 
-export default {
+const meta = {
 	title: 'BlockEditor/TextDecorationControl',
 	component: TextDecorationControl,
+	parameters: {
+		docs: {
+			canvas: { sourceState: 'shown' },
+			description: {
+				component: 'Control to facilitate text decoration selections.',
+			},
+		},
+	},
 	argTypes: {
-		onChange: { action: 'onChange' },
+		value: {
+			control: { type: null },
+			description: 'Currently selected text decoration.',
+			table: {
+				type: {
+					summary: 'string',
+				},
+				defaultValue: { summary: 'none' },
```

Comment:
```suggestion
				},
```

I think this is a mistake in the README. [The default value is undefined](https://github.com/WordPress/gutenberg/blob/2ea929cef1d225d4a075269453c5f570b874dddb/packages/block-editor/src/components/text-decoration-control/index.js#L45), not the string `"none"`.

Besides this part, could you also remove this line below?

https://github.com/WordPress/gutenberg/blame/widgets/fix/inserter-open/packages/block-editor/src/components/text-decoration-control/README.md#L31

Reply:
Sure, @t-hamano I have made changes to the README.md as well.